### ISSUE TITLE
logging custom loss

### ISSUE THREAD

**Issue by Ori226** [keras-team/keras#12883]
I want to log custom loss and visualize it on Tensorboard. I'm adding the loss by the model.add_loss(...) function. I can't add it as custom metrics since the model is not a classical "y_true, y_pred" type. It is more similar to Cycle GAN.

Any idea?

**Comment by psschand** [comment]
@Ori226 you can plot the loss — using custom callback function and using TensorBoard

I hope this [link ](https://www.codementor.io/blog/tensorboard-integration-5fh168wqvi)will be helpful

**Comment by Ori226** [comment]
thanks @psschand  but what I'm looking for is the custom loss. I don't have trouble with plotting the regular loss.

**Comment by ymodak** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.
To ask questions, please see the following resources :
[StackOverflow](http://stackoverflow.com/questions/tagged/keras)
[Keras Slack channel](https://keras-slack-autojoin.herokuapp.com)
[Keras.io support](https://keras.io/#support)
[Gitter](https://gitter.im/Keras-io/Lobby)
Thanks!
If you think we made a mistake, please open another issue explaining your request in detail.
